### ISSUE TITLE
There is no turrets to defend waves with

### ISSUE THREAD

**Issue by XtarsAgency** [Yoru-Kitsune/Tantros#1]
There is no turrets in the mod, and playing with "Tantros Caldera" map is impossible with the extremely low core health.
You can try to call me for it, like adding or spriting a new turret if you wanted to..

**Comment by CoCo-R38** [comment]
The mod is in very early access and I will make turrets and walls as one of the first things to do in the future... If you want to play the mod please play without waves for now... if you want to help please contact me on discord (yoru_kitsu) and there we can talk about further details

**Comment by CoCo-R38** [comment]
The reason why there are no turrets is mainly because I really care about the design and texture in my mods... and the turret has multiple sprites and the standard ressources aren't even completed yet so I think the standard ressources have a much higher priority than any waves and stuff

**Comment by XtarsAgency** [comment]
I dont have discord. I could be your spriter, i have done multiple things,
including the first mindustry mod made on android (Zeridmuth: A new
beginning)

Pada tanggal Sel, 12 Sep 2023 8.04 PM, Yoru ***@***.***>
menulis:

**Comment by CoCo-R38** [comment]
You're mod is not the first mod made on android... I can say this as I make my mods on android too and I develop mindustry mods for over a year now... I will look at the mods sprites tho and then we see if you are good enough for my mods spites

**Comment by XtarsAgency** [comment]
If you were to make sprites but unable to make much of them that fast, i can make them. Just like some with the mod "Netronium" which end up making a mod my own which is "Zeridmuth"

**Comment by XtarsAgency** [comment]
Sure. If you do accept i will do a Fork of your repository.
I primarily do this because i have a thought that this mod could in fact be
the true tantros mod that DOES get updated weekly or monthly, because its
mainly coded in HJSON not Java

Pada tanggal Sel, 12 Sep 2023 8.12 PM, Yoru ***@***.***>
menulis:

**Comment by CoCo-R38** [comment]
Sure... I looked at the sprites and some are probably not as intended but they are actually not that bad... if you give me the permission to modify your spites a bit so I can make sure they fit into the style
It would be good tho if you would download discord or at least make an account and chat with me on the browser bc it is definitly easier than github and I can quickly give you feedback and also it is easier to give you the materials you need and stuff like the basics of how I would like tomhave the spites... plus we could cooperate further as we aren't bound to any issue comments...

**Comment by XtarsAgency** [comment]
Yes, you do have permission to use my sprites, because the mod is actually
a random mod i decided to make :P

I could, but maybe sometime, i was hoping to maybe use my 32GB sd card to
use to download discord with


Pada tanggal Sel, 12 Sep 2023 8.26 PM, Yoru ***@***.***>
menulis:

**Comment by CoCo-R38** [comment]
I meant to modify your spites that you want to make for the mod... the sprites in your mod are good but they don't fit in the mod if you know what I mean... the sprites in the Tantros mod are depending on the tier they should be used on... I use different color palletes therefor and that's something that would be easier to discuss on discord... if you download discord just send you're username in here so I can add you and I can make sure that you are the one that texts with me

**Comment by XtarsAgency** [comment]
Alright, sure. I will get into you shortly after like around 2-3 days, i
have some homework to do (yes im still a student exactly) but dont worry, i
will not forget this thing, on a few days you can email me again or comment
me on github, if i can do it and if my homework doesnt pile up like that.
Anyway, my username would be "FomahaultC_Xtars", its still down because i
have some issues with my emails and stuff. See you later in a few days

Pada tanggal Sel, 12 Sep 2023 8.48 PM, Yoru ***@***.***>
menulis:

**Comment by CoCo-R38** [comment]
Okay! I'm a student too so don't worry... I know this

**Comment by XtarsAgency** [comment]
Alright. This is the last message i will send, i will need to go to sleep
(indonesian timezone in a nutshell) you can maybe check me up again after
sometime. Maybe 8-9 hours from now, either after i woke up and using my
phone in school or maybe freetimes. See you tomorrow

Pada tanggal Sel, 12 Sep 2023 8.59 PM, Yoru ***@***.***>
menulis:

**Comment by CoCo-R38** [comment]
I need to go to sleep in 8 hours-

**Comment by XtarsAgency** [comment]
Ok, no problem. This is the last message i will reply to, dont worry i can
be awake for some time tomorrow so i could do it

Pada tanggal Sel, 12 Sep 2023 9.08 PM, Yoru ***@***.***>
menulis:

**Comment by XtarsAgency** [comment]
Hello, i have wake up from sleep.
Can you tell me what is your discord name is so we can talk there?

Pada tanggal Sel, 12 Sep 2023 9.08 PM, Yoru ***@***.***>
menulis:

**Comment by XtarsAgency** [comment]
Ello?
